gradle/libs.versions.toml

```diff
@@ -29,9 +29,9 @@ androidx-test-junit = "1.3.0"
 androidx-window = "1.5.0"
 androidx-window-core = "1.5.0"
 androidx-window-java = "1.5.0"
-androidx-xr-arcore = "1.0.0-alpha07"
-androidx-xr-compose = "1.0.0-alpha08"
-androidx-xr-scenecore = "1.0.0-alpha08"
+androidx-xr-arcore = "1.0.0-alpha09"
+androidx-xr-compose = "1.0.0-alpha09"
+androidx-xr-scenecore = "1.0.0-alpha10"
 androidxHiltNavigationCompose = "1.3.0"
 appcompat = "1.7.1"
 arcorePlayServices = "1.0.0-alpha08"
```

Comment:
Please update to alpha09 as per https://maven.google.com/web/index.html?q=.xr.#androidx.xr.arcore:arcore-play-services